pages/main/faq.md

```diff
@@ -7,7 +7,43 @@ layout: documentation
 
 This page contains some Frequently Asked Questions.
 ## What is AsteroidOS?
-AsteroidOS is an open source Linux distribution that runs on many different smartwatches. It uses [Qt](http://www.qt.io/) and QML to provide the graphical interface.
+AsteroidOS is an open source Linux distribution that runs on many different smartwatches. It uses [Qt](http://www.qt.io/) and QML to provide the graphical interface.\
+One of the main goals of AsteroidOS is to provide users with greater control over their devices and data privacy. The operating system is fully customizable, and users can modify it to meet their specific needs.
+
+## Which features does AsteroidOS provide?
+Currently, AsteroidOS has these features available:
+ - phone notifications
+ - always on display
+ - tilt to wake
+ - palm to sleep
+ - multiple launcher styles
+ - (live) wallpapers
```

Comment:
I'm not just sure what "(live) wallpapers" means.

Reply:
Its ment to highlight the shader/animation part. But since it raises questions, just wallpapers will suffice!